src/components/Navbar.jsx

```diff
@@ -5,6 +5,8 @@ function Navbar() {
     <nav>
       <Link to="/">Home</Link>{"   "} 
```

Comment:
Ideally you shouldn't be relying on whitespace ({"     "}) to position your elements, you should be  using CSS to do this kind of styling when you do come to it